From 525f12b7ad72f4f6143df241ae5a77d731757bc3 Mon Sep 17 00:00:00 2001
From: Chris Fiege <chris@tinyhost.de>
Date: Sat, 19 Oct 2024 12:21:41 +0200
Subject: [PATCH 25/38] WIP: gluon-modules

modules: version bump onto v2021.1.x

modulfix

modules: update gluon packages to pick the ebtables-tiny locking fix

packages: Bump version to include: 'ebtables-tiny: reduce wait on missisng lock'

modules: Bump gluon-packages to v2023.2.x-ffbs-next

This drops patches for uradvd. This package has been moved to
openwrt-packages and will be re-added in a later commit.

modules: Move openwrt-packages to an ffbs-fork including patched uradvd

* 8a7f020cd WIP: Build uradvd from ffbs fork

modules: Bump gluon packages onto v2023.2.2

This bumps ffbs/packages.git to a version rebased onto what gluon uses
in v2023.2.2.

New commits in packages since v2023.2(.0):
* 3d08b0f wgpeerselector: fix undefined variable peer

modules: Bump openwrt packages onto gluon v2023.2.2

This bumps ffbs/openwrt-packages.git to a version rebased onto what gluon uses
in v2023.2.2.

New commits in packages since v2023.2(.0):

* 229175a0b libwslay: remove, nothing depends on it since h2o is gone
* b1b33235b dnsdist: disable XSK to fix the build
* 15ab42c47 dnsdist: update to 1.9.0
* 14c9d6172 dnsdist: Enable custom load-balancing policies in the light version
* c55605659 dnsdist: update to 1.8.3
* 218769f99 h2o: remove, nothing depends on it anymore
* 1504a153b nghttp3: fix pkgconfig file
* c2350acb6 clamav: update to 1.3.0
* fe210e7cb knot: update to version 3.3.5
* eaa6026dd acme-common: use validation_method option instead of guessing
* 6c7b6bf20 ntpclient: remove
* 06fb98b27 syslog-ng: update to version 4.6.0
* 002297b47 syslog-ng: conf: fix deprecated stats_freq
* fd9ccec73 v2ray-core: Update to 5.14.1
* 210468d0c c-ares: update to 1.27.0
* af4d84746 apfree-wifidog: update to 7.02.1977
* f629091e8 apfree-wifidog: fix some bugs in the wifidogx.init file
* 817246f59 Merge pull request #23573 from stangri/openwrt-23.05-https-dns-proxy
* 667ac9d8e banip: update 0.9.4-3
* 30cd11d03 natmap: update to 20240303
* 81c1d4449 v2raya: Update to 2.2.5.1
* 4778acec8 dnsproxy: Update to 0.65.2
* 244320826 golang: Update to 1.21.7
* 8d38305fa v2raya: Update to 2.2.5
* ac96098e3 banip: update 0.9.4-2
* 2e4d1635d banip: release 0.9.4-1
* 4442726ba rclone: add fuse3-utils as dependency
* e3521f16c yq: Update to 4.42.1
* cadd900f0 v2fly-geodata: Update to latest version
* 44febef1c cloudflared: Update to 2024.2.1
* 9e20e91de btop: Update to 1.3.2
* 751c3a580 v2raya: Update to 2.2.4.7
* 00f32a022 yq: Update to 4.41.1
* 5e5e0592d yq: Update to 4.40.7
* 9b57a8a68 cloudflared: Update to 2024.2.0
* 0d013a38b v2ray-geodata: Update to latest version
* 7bc7eb58f https-dns-proxy: update to upstream 2023-11-19
* 3c3477267 rtl-sdr: update to v2.0.1
* b6f2bef08 rtl_433: update to 23.11
* 5ecbe0797 travelmate: update 2.1.2-5
* 166e8d2d3 adblock: update 4.1.5-9
* ab651824e ragel: new package to build vectorscan
* cd5f6637f snort3: build against gperftools-runtime
* 525321667 snort3: update to 3.1.81.0
* 28aeff87b gperftools: add new package
* f589b5880 hyperscan: new package for speeding up regex ops
* c9dff69de dockerd: Update to 25.0.3
* 755e6f783 docker: Update to 25.0.3
* 984b65bf7 containerd: Update to 1.7.13
* a697eea62 runc: Update to 1.1.12
* b3270866f tailscale: Update to 1.58.2
* bd5a36867 tailscale: Update to 1.58.0
* 1ff74e644 travelmate: update 2.1.2-4
* fa68279e3 Merge pull request #23485 from mhei/23.05-php8-update-to-8.2.16
* e5c557be0 travelmate: update 2.1.2-3
* 309ee2c26 git: update to 2.43.2
* 53fa23900 git: update to 2.43.0
* f628aaa30 lighttpd: update to lighttpd 1.4.74 release hash
* 06b0d0cce ovn: bump to 22.03.5
* 0000f307c openvswitch: bump to 2.17.9
* 65883df9a squid: fix configure options
* 57acafa66 unbound: update to latest upstream release version 1.19.1
* 76aaa3059 libuv: fix CVE-2024-24806
* 1a4c42d69 haproxy: update to v2.8.6
* b23808a54 ocserv: updated config
* 8e9ad0e3c ocserv: use better separator for sed
* a19f8a1f1 openconnect: make host dependency more resilient
* 22e304ecf openconnect: update to 9.12
* 679b9cf55 openconnect: add support for option --pfs
* 3d7193cde php8: update to 8.2.16
* 9164fa255 lxc: update to 5.0.3
* c5ad47372 zabbix: update to version 6.4.7
* 583af1248 yt-dlp: bump to version 2023.12.30
* abba8cff8 yt-dlp: bump to version 2023.11.16
* a696cd44b yt-dlp: add missing dependencies
* f12547cf1 node: February 14 2024 Security Releases
* 9d8882fe4 Merge pull request #23407 from stangri/openwrt-23.05-adblock-fast
* 95f45c24b bind: bump to 9.18.24
* 1159b8ec3 bind: bump to 9.18.19
* 74cd5e095 bind: update to version 9.18.18
* dc3fec89a bind: bump to 9.18.16
* 2451d5a69 pdns-recursor: update to 4.8.6 (fixes CVE-2023-50387, CVE-2023-50868)
* ae10fd6db Merge pull request #23415 from systemcrash/p910nd_picks
* 836047068 squid: update to 6.7
* f711c7e24 p910nd: bump release
* 0f0962ee1 p910nd: hotplug shellcheck fixes
* 1e8c78fcc p910nd: init: check device (/dev/usb/lpX) existence
* 111228675 p910nd: init: partial fix for openwrt/packages#10496
* 238e16f97 p910nd: hotplug+init: include extra ieee1284 properties
* 00d14e9bb p910nd: hotplug: minor bug fixes
* b7a11d611 p910nd: hotplug: small refactor
* 92d1d4a77 p910nd: init: add txtvers=1 to mDNS properties
* c847cac8f p910nd: init: line-break and conditionalize mDNS properties
* 9ea1d672b p910nd: init: only run mDNS changes if mdns is set to on
* 7eeb756a2 p910nd: init script
* 3cac4412d p910nd: hotplug script
* abe112566 adblock-fast: add force_dns_interface setting
* 280b81c7a stlink: add packages
* d844df9a4 crowdsec: new upstream release version 1.6.0
* 91c5ca7ba openssh: fix build failure on powerpc_8548 https://github.com/openssh/openssh-portable/commit/1036d77b34a5fa15e56f516b81b9928006848cbd
* 89fcc9e27 openssh: bump to 9.6p1
* fad7e0ae1 tailscale: create combined tailscale/tailscaled
* cf0b1e0ed knot: update to version 3.3.4
* de34798be knot: update to version 3.3.3
* f83e01a4b dawn: Add PKG_BUILD_FLAGS:=no-lto because lto causes strange SegFaults
* a062c2a51 dawn: Update to 2023-05-14
* dbb40a890 snowflake: update to 2.8.1
* 26c24e579 mosquitto: reenable options wrongly turned off
* db28d99be python-ble2mqtt: update to 0.2.2
* c2ae7b95d python-dbus-fast: upgrade to 2.21.1
* 63f6af4bc nextdns: Update to version 1.42.0
* 6d6e398c7 Openvpn: add missing script-security
* bce14a016 cloudflared: refine config.yml
* 892ce0645 dnsproxy: Update to 0.64.1
* 3ffac159c inih: Update to r58
* a3c2cd8fb rclone: Update to 1.65.2
* 8c8ec8106 cloudflared: Update to 2024.1.5
* 6cb714722 dos2unix: Update to 7.5.2
* 05c602c8c Merge pull request #23253 from stangri/openwrt-23.05-curl
* ff387bd30 Merge remote-tracking branch 'yggdrasil-openwrt/yggdrasil-for-23.05' into openwrt-23.05
* 8d02881d4 curl: update to 8.6.0
* b2bf8f26b Merge pull request #23227 from stangri/openwrt-23.05-nebula
* dcff607ba nebula: update to 1.8.2-2
* 810bd106d Merge pull request #23201 from mhei/23.05-php8-update-to-8.2.15
* 905afb0ed xz: Update to 5.4.6
* 92a07db7d php8: update to 8.2.15
* 1167ef72e python-dbus-fast: update version 2.21.0
* e6d3a7892 python3-bleak: add missing python3-typing-extensions dependency
* 0da0a501e natmap: update to 20240126
* 67dcaa5d7 sing-box: update to 1.8.4
* f87af4985  banip: update 0.9.3-5
* b61930ad7 v2ray-geodata: Update to latest version
* 9a26ff67e v2raya: do not allow changing config/log directory
* 4267c8776 dnsproxy: Update to 0.63.1
* 63a022a87 rclone: Update to 1.65.1
* d07aa2f0f xray-core: Update to 1.8.7
* d9d4acfd9 cloudflared: Update to 2024.1.4
* ebddc07c1 cloudflared: Update to 2024.1.2
* ef606d4b2 Merge pull request #23137 from nmav/tmp-new-ocserv-23.05
* 4da835d3e gnutls: updated to 3.8.3
* 5aabddf75 ocserv: updated to 1.2.4
* 564e6e08d Openvpn: add missing script event options
* 0707d98c1 htop: update to 3.3.0
* d877e4493 Merge pull request #23127 from stangri/openwrt-23.05-adblock-fast
* ac2736542 adblock-fast: update to 1.1.1-3
* 2c2bc1709 Merge pull request #23117 from stangri/openwrt-23.05-adblock-fast
* 58a7c28aa adblock-fast: sync with main branch
* 144cacc12 Merge pull request #23101 from stangri/openwrt-23.05-nebula
* ee11377b7 nebula: update to 1.8.2
* eb7db48b6 yggdrasil: bump to 0.5.4
* 315f5c41b yggdrasil: bump to 0.5.3
* be2b05436 yggdrasil: support public key in config generation
* 4b5d833e6 yggdrasil: overhaul package with netifd support - package is bumped to 0.5.2 - new protocol changes prevent peering with 0.4.x peers - @turretkeeper revamps package with netifd support - do not use with luci-app-yggdrasil please install luci-proto-yggdrasil

modules: OpenWRT-packages onto v2023.2.3

Out fork of openwrt-packages has been rebased onto what Gluon v2023.2.3
references.

New commits:

* 3db4314e2 delve: update to 1.23.0
* 6e2e3240d xray-core: update to 1.8.21
* c2422e740 p910nd: set bidi only if not already set
* a8c75f103 make: update to 4.4.1
* 3016416c5 autoconf: update to 2.72
* bf11148e5 automake: update to 1.16.5
* 2d98802b1 xray-core: update to 1.8.20
* 66e793f9c unbound: Update to 1.20.0
* 7ebcd8c81 transmission: update to version 4.0.6
* 733e6808f transmission: fix compile with MbedTLS 3.X
* 16446a3b2 bind: bump to 9.18.27
* dbac39060 ddns-scripts: Update knot resolver regexp
* f2044c19c exim: update to 4.98
* 1b51b17c7 xray-core: update to 1.8.19
* 89955ee71 Merge pull request #24572 from mhei/23.05-php8-update-to-8.2.21
* d14e460b5 dockerd: fix breaks IPv6 routing
* 8403d1466 v2ray-geodata: Update to latest version
* 7e77e583f v2ray-geodata: Update to latest version
* 2ac9aba4e v2ray-geodata: Update to latest version
* c7cebd6ca v2raya: update to 2.2.5.7
* 08e499044 xray-core: update to 1.8.18
* dc6172ecc banip: update 1.0.0-5
* 46c43fc4b krb5: update to 1.21.3
* 62fffe1a4 xray-core: update to 1.8.17
* 658aeee10 dnsproxy: Update to 0.67.0
* 05a0fbe21 php8: update to 8.2.21
* 37a65f217 adguardhome: Update to 0.107.46
* 9042b657d adguardhome: Update to 0.107.42
* 6908bf51a adguardhome: wait for interfaces to be up at boot
* 62febf216 Merge pull request #24546 from ynezz/ynezz/openwrt-23.05/license-fixes-backports
* b7665f4f6 v2raya: update to 2.2.5.6
* f19d10782 v2raya: update to 2.2.5.5
* b092cb779 apfree-wifidog: Update to 7.07.2018
* da130e8bb apfree-wifidog: support wildcard domain
* f24f16576 lang/python/python-paho-mqtt: fix license
* 8b613973b net/ntpd: fix license
* e6e5f761b utils/lsof: fix license
* e24386b3e libs/libgd: fix license
* f0780bb91 libs/tiff: fix license
* 92a4808a2 libs/libev: fix license
* e2e463171 net/tor: add license
* 652cce0ed devel/automake: fix license
* a4d7058fe utils/vim: add license info
* 0d4d322e6 libassuan: Fix license
* 10babb22c gpgme: Fix license
* cf2ef8337 node: July 8, 2024 Security Releases
* e1b996e98 golang: Update to 1.21.12
* e56f8ebef sms-tool: version bump to 2023-09-21
* 6d08bdf37 rust: select correct architecture for armv5
* a0bf47605 rust: add patch to fix remote filesystem issue
* e081cb182 rust: Update to 1.78.0
* 35b740e4e jq: Add PKG_CPE_ID
* 32c5137a1 uwsgi: Add PKG_CPE_ID
* 4fe6b2c67 mxml: Add PKG_CPE_ID
* 8746e3a3d net/iperf: assign PKG_CPE_ID
* 367b77884 net/iperf3: assign PKG_CPE_ID
* b493b3e5b net/i2pd: assign PKG_CPE_ID
* 557fc52d6 libs/libvpx: fix PKG_CPE_ID
* d31e6b044 libs/apr: fix PKG_CPE_ID
* 73c3c2bac libs/libestr: assign PKG_CPE_ID
* f0195d89d net/tcpreplay: fix PKG_CPE_ID
* 22d6d7132 utils/cryptsetup: assign PKG_CPE_ID
* 3b9d7313b libs/fmtlib: assign PKG_CPE_ID
* a490b7063 net/atftp: assign PKG_CPE_ID
* 990440705 libs/libyang: assign PKG_CPE_ID
* ff916f3d3 multimedia/gst1-plugins-bad: assign PKG_CPE_ID
* b1fd4362f lang/vala: assign PKG_CPE_ID
* 6d8a44764 libs/freetype: fix PKG_CPE_ID
* 1535d8ca8 utils/domoticz: assign PKG_CPE_ID
* 8c7241778 libs/libinput: assign PKG_CPE_ID
* 2af503f31 net/fail2ban: assign PKG_CPE_ID
* 9131151ef utils/crun: assign PKG_CPE_ID
* 9f3b98ae0 utils/pax-utils: assign PKG_CPE_ID
* 5934d6908 lang/python/python-docker: assign PKG_CPE_ID
* 3521fa98f admin/atop: assign PKG_CPE_ID
* 2e12f6109 libs/libmbim: assign PKG_CPE_ID
* f878002e9 net/fastd: assign PKG_CPE_ID
* 5e947756d utils/augeas: assign PKG_CPE_ID
* b56db1542 utils/cgroupfs-mount: assign PKG_CPE_ID
* cd78db517 utils/acl: assign PKG_CPE_ID
* 029aede36 net/bwm-ng: assign PKG_CPE_ID
* 0d8817c58 libs/libgee: assign PKG_CPE_ID
* 72f0ab05e libs/libpam: fix PKG_CPE_ID
* dc88a5923 net/krb5: fix PKG_CPE_ID
* 4fc01ae35 libs/vips: fix PKG_CPE_ID
* 4fcfa3e9b admin/sudo: fix PKG_CPE_ID
* a8345ffad utils/squashfs-tools: fix PKG_CPE_ID
* 8bb8ac555 lang/python/python-aiohttp: fix PKG_CPE_ID
* 2f0211988 lang/python/python-urllib3: fix PKG_CPE_ID
* 9c14028de lang/python/python-requests: fix PKG_CPE_ID
* e32911eaf libs/redis: fix PKG_CPE_ID
* c89911693 net/vsftpd: fix PKG_CPE_ID
* 50f8f6aea net/tinc: fix PKG_CPE_ID
* cd311ffe1 net/tinyproxy: fix PKG_CPE_ID
* e7714d692 utils/tmux: fix PKG_CPE_ID
* f04d71055 utils/zsh: fix PKG_CPE_ID
* 504faf4ba net/boinc: fix PKG_CPE_ID
* 5c1e4ca0e utils/lrzsz: fix PKG_CPE_ID
* df3b0d47e libs/expat: fix PKG_CPE_ID
* ca577d40e libs/libidn2: fix PKG_CPE_ID
* 19c07a2ef net/miniupnpc: fix PKG_CPE_ID
* eea8ccef9 multimedia/motion: fix PKG_CPE_ID
* 888418531 net/nbd: fix PKG_CPE_ID
* 866f4dabc lang/python/python-pip: fix PKG_CPE_ID
* af3ae451d utils/gpsd: fix PKG_CPE_ID
* 8a13733a2 utils/ntfs-3g: fix PKG_CPE_ID
* 84d7eb7af treewide: assign PKG_CPE_ID
* 19dd98e9f lang/python/python-yaml: fix PKG_CPE_ID
* bbb20bd57 treewide: fix licence typos
* a9ee98f9e openssh: bump to 9.8p1
* 8d8f0b461 dockerd: Update to 27.0.3
* 5a013e8d1 docker: Update to 27.0.3
* 07003b75d docker: Update to 27.0.2
* 532f64e5b runc: Update to 1.1.13
* d39340140 dockerd: Update to 27.0.2
* e61095661 dockerd: Update to 26.1.4
* 1660023b9 containerd: Update to 1.7.18
* c8c946ef1 docker: Update to 26.1.4
* da532f52c adblock: update 4.1.5-11
* cac8cba09 jool: update to 4.1.12
* 86cd13e81 jool: update documentation
* 1de4596df jool: update to 4.1.11
* 286a830c3 jool: update package to version 4.1.10
* 55ef99398 cloudflared: Update to 2024.4.1
* 341665c75 cloudflared: Fix incorrect uci config syntax
* f01fbf891 cloudflared: Add more run parameters in UCI
* 39b307f1d adblock: update 4.1.5-10
* 5747af1a5 aardvark-dns: update to 1.11.0
* c2f35ddc2 aardvark-dns: update to 1.10.0
* f47158a63 libjwt: add package
* 153984a8f powertop: update to 2.15
* 6502d4ca2 xray-core: update to 1.8.16
* 7bd5f1dc2 xray-core: update to 1.8.15
* 62f50a26d natmap: reset PKG_RELEASE to 1
* 3a0073c4d natmap: update to 20240603
* 15c2051ea banip: update 1.0.0-4
* 44231285d golang: Update to 1.21.11
* 4eb73df5e lighttpd: fix missing dependency for OpenSSL crypto library
* 0302f8cf6 lighttpd: add option to use OpenSSL crypto library
* 8e683037f schroot: fix compilation with GCC14
* 2cfcb0893 stlink: fix compilation with GCC 14
* d25180284 prometheus-node-exporter-lua: Add optional mwan3 collector
* 87a831fa8 prometheus-node-exporter-lua: Add "node_textfile_mtime_seconds" metric
* 611b37244 prometheus-node-exporter-lua: remove duplicated nat samples
* 2fbca34c7 prometheus-node-exporter-lua: fix netclass duplicate TYPE lines
* 49ff5da74 prometheus-node-exporter-lua-hostapd_stations: fix not reporting metrics
* 8cb0a16d1 prometheus-node-exporter-lua: add missing libubus-lua dependency
* d1db488d9 prometheus-node-exporter-lua: bump package version
* fc1b9071a prometheus-node-exporter-lua: add basic hwmon exporter
* 7b683fd65 prometheus-node-exporter-lua: Add thermal collector
* 464ca5325 banip: update 1.0.0-3
* 6d81b33c8 apfree-wifidog: update to 7.06.2008
* 0e098b8fe apfree-wifidog: support rule group and websocket&dns proxy flag
* 287d5b865 ruby: update to 3.2.4
* 2919fae48 banip: update 1.0.0-2
* 76bb48944 sing-box: update to 1.9.3
* 6da721a4c sing-box: update to 1.9.2
* 564866110 sing-box: update to 1.9.1
* e0d786351 unbound: pull in adblock-fast generated adb_list
* 36cf3cfb3 Merge pull request #24340 from mhei/23.05-php8-update-to-8.2.20
* 6be3892dc iperf3: fix usage with big endian
* 98811c336 iperf3: update to 3.17.1
* c9829c92e banip: release 1.0
* 03e0dd76c php8: update to 8.2.20
* a18384954 libpfring: backport patch fixing compilation error for sa_data
* 29b1b8301 libpfring: update to 8.4.0
* 0f0c24a81 hev-socks5-server: update to 2.6.6
* d36268241 Merge pull request #24317 from p-w-p/xray-core_update
* d81d04aaf xray-core: update to 1.8.13
* 5ad306413 node: bump to v18.20.3
* b131f8ef8 sing-box: update to 1.9.0
* 37dc80403 banip: update 0.9.6-3
* 93b43c2b9 apache: add compile fix for libxml-2.12.x
* e621c8f67 banip: update 0.9.6-2
* e5a9ac21e xfrpc: update to 3.05.661
* 123ee5222 xfrpc: Revised the config file and adjusted the corresponding init file
* 57d3da509 ocserv: use a more neutral character for sed
* 221949b6c nextdns: Update to version 1.43.5
* dbcbbb27e sing-box: update to 1.8.14
* 64d2e702f dnsdist: update to 1.9.4
* 20f58e465 haproxy: update to v2.8.9
* d8cb9f1af banip: release 0.9.6-1
* a14bb12df openconnect: introduced URI parameter
* 233405f68 openconnect: backport fix for anyconnect compatibility
* 913cda459 gptfdisk: update to 1.0.10
* 2da338d5c golang: Update to 1.21.10
* 00cf7e87a Merge pull request #24139 from stangri/openwrt-23.05-adblock-fast
* 4369aa026 pdns-recursor: update to 4.8.8
* c230bf75a adblock-fast: update to 1.1.2-1
* ade006dc9 ocserv: updated to 1.3.0
* e983686bf banip: update 0.9.5-5
* 9527d8f02 docker: Update to 26.1.0 * Removed unnecessary GO lang variables
* 484cd2a93 dockerd: Update to 26.1.0 * Removed unnecessary GO lang variables
* fad630db7 containerd: Update to 1.7.15 * Explicitly list GO_PKG_INSTALL_EXTRA * Removed unnecessary GO lang variables
* 16b5d1072 uspot: update to Git HEAD (2024-05-03)
* 4702db286 xray-core: Update to 1.8.11
* f47b6403c xray-core: Update to 1.8.10
* 525643f83 xray-core: Update to 1.8.9
* f61012a01 v2ray-core: Update to 5.16.0
* 4bc116560 v2ray-geodata: Update to latest version
* 798738c9b acme-acmesh: use validation_method option instead of guessing
* ff8429b3c hev-socks5-server: add new package
* e525a3f0f nano: update to 8.0
* 9aa2479ca banip: update 0.9.5-4
* 27488be21 nextdns: Update to version 1.43.3
* 2955c2db8 dnsproxy: add three new features
* 7aec92689 msmtp: update to version 1.8.25
* 7492e8cc9 transmission: update to version 4.0.5
* 94fca3f1f sing-box: update to 1.8.12
* f00ce8a58 Merge pull request #24023 from rs/nextdns-1.43.0-openwrt-23.05
* f9e15a7b0 nextdns: Update to version 1.43.0
* 24e2a32bd libndpi: backport patch for PCRE2 support
* 988181e6a openssh: bump to 9.7p1
* 4854a5983 librespeed-go: improve the description
* cbb2e66cc librespeed-go: Reload the daemon after modifying the tls certificate
* ca4569ead v2ray-core: Update to 5.15.3
* 4979ade99 banip: update 0.9.5-3
* a08e087dc banip: update 0.9.5-2
* a4732b76b banip: release 0.9.5-1
* 34a650bba syslog-ng: update to version 4.7.1
* 12adb901d CI: remove CircleCI for now
* ea65457f1 Merge pull request #24014 from stangri/openwrt-23.05-adblock-fast
* 21efddca3 adblock-fast: bugfix: unbound-related fixes
* 3d63a10cb Merge pull request #24006 from stangri/openwrt-23.05-nebula
* 4e6eb8f5b nebula: Use APK style release number
* d51ad546b natmap: add log_std{out,err} options
* fd4899e7b node: bump to v18.20.2
* 4fac0d532 ntpd: update to version 4.2.8p17
* 0f82c9b85 uwsgi: bump to latest 2.0.25.1 release
* 86b9791db uwsgi: add experimental pcre2 patch and drop pcre
* d5b7ea545 uwsgi: bump to release 2.0.22
* 7414685bf v2ray-geodata: Update to latest version
* 9967432bd v2ray-geodata: Update to latest version
* 7286ea81b cloudflared: Update to 2024.4.0
* 9caf90f75 tor: update to 0.4.8.10 stable
* e1ae68ee3 Merge pull request #23947 from mhei/23.05-php8-update-to-8.2.18
* b301e8c28 Merge pull request #23871 from graysky2/snort-backport-fix
* 8940a6256 exim: update to 4.97.1
* 51ce5a553 cryptsetup: update to version 2.7.1
* ac9096d32 lvm2: update to LVM2 2.03.17 and libdm Version 1.02.187
* 17cac25f9 gnunet: update to version v0.21.0
* f0368a249 gnunet: update to version 0.20.0
* 181e09b7d gnunet: update to version 0.19.4
* 2f1f27d47 libcurl-gnutls: update to verison 8.7.1
* d5ae95aa1 libcurl-gnutls: fix build
* d291474db libcurl-gnutls: update to version 8.6.0
* 2f4aa5376 libcurl-gnutls: update to version 8.5.0
* 7c3d2ef63 libcurl-gnutls: update to version 8.2.1
* f1e6365bc php8: update to 8.2.18
* 6dfe66fd8 golang: Update to 1.21.9
* 8fb4719f2 lighttpd: update to lighttpd 1.4.76 release hash
* 802b4c3af Merge pull request #23874 from stangri/openwrt-23.05-adblock-fast
* d742be207 Merge pull request #23815 from stangri/openwrt-23.05-curl
* e6fde07dd lualanes: update to version 3.16.3 and use tarball
* 1a9c3d093 lualanes: Version bump to v3.16.2
* 68421b9a2 hwdata: update to 0.379
* f7251c4f8 libs/libdaq3: assign PKG_LICENSE_FILES
* e1f0df951 libdaq3: update to 3.0.14
* 4acf815dd libdaq3: update to 3.0.13
* 19c641a26 libdaq3: update to 3.0.11
* 5596fa277 snort3: remove symbol @HAS_LUAJIT_ARCH
* 4bbd27065 snort3: update to 3.1.82.0
* aef7a4a8e adblock-fast: improve Makefile's prerm
* 8602f63b8 node: April 3, 2024 Security Releases
* 76bc13be7 irqbalance: update to version 1.9.4
* 39b8d5c7a irqbalance Update init script to remove duplicate spaces
* d9d6eb1d8 nano: fix syntax highlighting for raw ucode scripts
* a3a26de00 nano: add syntax highlighting for ucode scripts
* 84bfbcb6c curl: update to 8.7.1
* 35773ec8c kmsbd-tools: switch to use tagged release
* c14e0805f ksmbd-tools: update to 3.5.1
* 876b0bc19 docker-compose: add PKG_NAME to PKG_SOURCE
* 66894032d acme: standardize key_type
* 015e212ad sing-box: update to 1.8.10
* b1363fa06 sing-box: restart if the specified interfaces start up
* 6c75a9dae sing-box: update to 1.8.7
* 73bee1f9b acme-common: backport config fixes from master
* a0ad76b2d rust: update to 1.77.0
* 6578b79b0 rust: update to 1.76.0
* cbf854218 uspot: update to Git HEAD (2024-03-25)
* 7ffb90e93 dnsproxy: Update to 0.66.0
* 096ef93f5 cloudflared: Update to 2024.3.0
* c781bc0d9 v2ray-geodata: Update to latest version
* f282cdf59 v2ray-core: Update to 5.15.1
* d51c6bb7a rclone: Update to 1.66.0
* 2d5f789cc p910nd: fix running multiple instances
* a49c7d20f golang: Update to 1.21.8
* 91628e80b syslog-ng: enable http module based on zlib support in curl
* 69870fc94 ci: set correct arch for rootfs tests
* 53896bd1a Merge pull request #23736 from stangri/openwrt-23.05-https-dns-proxy
* 2423a6e93 https-dns-proxy: prepare migration to APK
* 283f7a06f Merge pull request #23719 from stangri/openwrt-23.05-https-dns-proxy
* 69910d2f4 Merge pull request #23720 from stangri/openwrt-23.05-adblock-fast
* e8cf63f09 Merge pull request #23718 from stangri/openwrt-23.05-nebula
* 95a40179f knot-resolver: Update to version 5.7.1
* 92bb77af4 knot-resolver: enable dnstap module build by default
* fd10fa17a knot-resolver: do not overwrite -Ddnstap=enabled configuration
* ed3cba66e lighttpd: update to lighttpd 1.4.75 release hash
* bc08d6e0e nebula: prepare migration to APK
* bdc9d4a5c https-dns-proxy: prepare migration to APK
* fcfac651c adblock-fast: prepare migration to APK
* 063b2393c django: bump to version 4.2.11
* 5782faa66 Merge pull request #23691 from EricLuehrsen/unbound_1193_wrt23
* ce8fb7aa9 unbound: update to 1.19.3
* d76cb891f unbound: spell fix
* 6d2657fc8 unbound: add file parameter to service instance
* e65ce6218 unbound: remove date/time from config headers
* b713fe7d2 travelmate: update 2.1.2-6
* 266846a36 p910nd: restart daemon even if no driver file is needed
* bc2c2cc29 shared-mime-info: update to 2.4
* 46595ddc0 shared-mime-info: update to 2.2
* 60ab78341 shadowsocks-libev: add remote server ips to dst bypass ipset

modules: Bump packages

This change is a no-op. The following commit has been rewritten:

* 953a6a9 ebtables-tiny: reduce wait on missisng lock

modules: Rebase onto v2023.2.4
---
 modules | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/modules b/modules
index d9813cf6..c7b3e167 100644
--- a/modules
+++ b/modules
@@ -4,12 +4,13 @@ OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-23.05
 OPENWRT_COMMIT=533235182141392d8031a6d2edb9d7edc91bfd5a
 
-PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
-PACKAGES_GLUON_COMMIT=3d08b0fee8dc5d96d8bcdb985fad1d5564de4022
+PACKAGES_GLUON_REPO=https://gitli.stratum0.org/ffbs/packages.git
+PACKAGES_GLUON_COMMIT=953a6a9dde028491e1eba4a85d4ca2ebbf3d1bc0
+PACKAGES_GLUON_BRANCH=v2023.2.x-ffbs-next
 
-PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
-PACKAGES_PACKAGES_BRANCH=openwrt-23.05
-PACKAGES_PACKAGES_COMMIT=33858483a03f322ec1233a98816c3d3197c177dd
+PACKAGES_PACKAGES_REPO=https://gitli.stratum0.org/ffbs/openwrt-packages.git
+PACKAGES_PACKAGES_BRANCH=openwrt-23.05-ffbs-next
+PACKAGES_PACKAGES_COMMIT=9c0bcfd73989e53136a78eeed114c1921ede64eb
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt/routing.git
 PACKAGES_ROUTING_BRANCH=openwrt-23.05
-- 
2.34.1

